tests/models/test_renderedtifields.py

```diff
@@ -96,28 +95,29 @@ def teardown_method(self):
         self.clean_db()
 
     @pytest.mark.parametrize(
-        "templated_field, expected_rendered_field",
+        ["templated_field", "expected_rendered_field"],
         [
-            (None, None),
-            ([], []),
-            ({}, {}),
-            ((), "()"),
-            (set(), "set()"),
-            ("test-string", "test-string"),
-            ({"foo": "bar"}, {"foo": "bar"}),
-            (("foo", "bar"), "('foo', 'bar')"),
-            ({"foo", "bar"}, "{'foo', 'bar'}"),
```

Comment:
Changed the set to being a single member, Eliminated the reason why it can lead to ordering issue!

We only want to check how set behaves: one empty and one with member! so we don't need two members

Reply:
Really smart. Basic sanity is important!